### ISSUE TITLE
Accessibility finding: Letter loading status is not announced to assistive technology

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#136991]
### Checklist item

WEB-413	Important updates are announced to screen readers automatically

### Severity level

2, High. Should be fixed in 1-2 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/records/download-va-letters/letters

### Details

When the accordion is opened, the loading indicator is not consistently announced to assistive technology. For example, it’s announced on iOS/Safari/VoiceOver but not on Mac Safari/VoiceOver. When loading completes and the indicator is replaced with a download link, that change is not announced on any AT. 

### Reproduction steps

1. Use Safari + VoiceOver on Mac and navigate to the "Foreign Medical Program Enrollment Letter" accordion button. 
2. Activate the button
3. The loading indicator appears and after loading is complete a Download link appears. You don't hear either change.

### Proposed solution or next steps

Both the loading indicator and the download link could go in an `aria-live="polite"` region. Maybe include some sort of message about the loading being complete as well.

Could also switch to the button download, used under "Benefit Summary and Service Verification Letter", but apparently that letter uses a button because we're actually making an API call, while the letters that use the link (including FMP) are linking to an already existing letter. 

Make sure fix works with https://github.com/department-of-veterans-affairs/va.gov-team/issues/136993

**Comment by jasonday** [comment]
Confirmed as issue.
